hawkbit-sdk/hawkbit-sdk-device/src/main/java/org/eclipse/hawkbit/sdk/device/SetupHelper.java

```diff
@@ -7,7 +7,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  */
-package org.eclipse.hawkbit.sdk.demo;
+package org.eclipse.hawkbit.sdk.device;
```

Comment:
Idea was to have everything you need to setup/work with devices in one module - sdk device,
meaning anything that needs to implement and work with devices to have everything there (This particular SetupHelper used inside DDiTenant so that one can work directly with its DDiTenant in order to setup the devices)